cms/src/helpers/uploadTemplate.js

```diff
@@ -170,3 +173,96 @@ export const createModelUploadTemplate = async authClient => {
     });
   });
 };
+
+export const createVariantUploadTemplate = async authClient => {
+  const variantColumnHeaders = [
+    'Model Name',
+    'Variant Name',
+    'Variant Type',
+    'Assesment Type ',
+    'Expression Level',
+  ];
+
+  const variantData = await Variant.find({});
+  const variantNames = variantData.map(variant => variant.name);
+
+  const sheets = google.sheets({ version: 'v4', auth: authClient });
+
+  // Date formatted like: May-29-2020-13:51:42
+  const dateStamp = Date()
+    .split(' ')
+    .splice(1, 4)
+    .join('-');
+
+  const spreadsheet = {
+    resource: {
+      properties: { title: `${dateStamp} HCMI Bulk Upload Variants Template` },
+      sheets: [
+        {
+          properties: { title: 'Variants', index: 0 },
+          data: {
+            rowData: [headerRowData(variantColumnHeaders)],
+          },
+        },
+      ],
+    },
+  };
+
+  const response = await new Promise((resolve, reject) => {
+    sheets.spreadsheets.create(spreadsheet, (err, data) => {
+      if (err) {
+        console.log('Error occured creating google sheet for models bulk upload template.');
```

Comment:
Not sure if this error message needs to mention variants specifically instead of models

Reply:
thanks. fixed.